### DISCUSSION TITLE
Get notified when device has disconnected

### DISCUSSION THREAD

**Discussion by ddppddpp** [Koenkk/zigbee2mqtt#17241] [Devices]
I have migrated most of my zigbee devices to z2m now and I'm quite happy with the powerful capabilities I got in the process, however sometimes a random device disconnects and I only get to find out when say a Home Assistant automation fails on me or a light doesn't turn on or whatever.
Is there a way to get a notification (i.e. published to an mqtt topic) when a device disappears from the radio network? Doesn't matter what the cause - bad connection, buggy device, drained battery... I just need to know when this happens so I can act and try to (manually) fix it.
Thx in adavnce!

**Comment by Busta999** [comment]
Now I am new to zigbee2MQTT and may have this wrong…

but in zigbee2MQTT settings/advanced you see an option Last Seen, that posts last seen time, also looks like it does other reporting to MQTT too.

hope that helps
